tests/manifests/mermaid/test_mermaid.py

```diff
@@ -0,0 +1,361 @@
+        d | r | b | m | property | type    | ref       | source      | prepare | access
+        datasets/gov/example     |         |           |             |         |
+          | data                 | sql     |           |             |         |
+                                 |         |           |             |         |
+          |   |   | Country      |         |           | salis       |         |
+          |   |   |   | name     | string  |           | pavadinimas |         | open
+          |   |   |   | id       | integer required|           | id          |         | open
+        '''))
+
+    cli.invoke(rc, [
+        'copy',
+        '--no-source',
+        '--access', 'open',
+        '-o', tmp_path / 'result.mmd',
+              tmp_path / 'manifest.csv',
+    ])
+
+    with open(tmp_path / 'result.mmd', "r") as file:
+        contents = file.read()
+        assert contents == """---
+datasets/gov/example
+---
+classDiagram
+class Country {
++ name : string [0..1]
++ id : integer [1..1]
+}
+"""
+
+
+def test_copy_mmd_array(context: Context, rc, cli: SpintaCliRunner, tmp_path):
+    create_tabular_manifest(context, tmp_path / 'manifest.csv', striptable('''
+        d | r | b | m | property | type    | ref       | source      | prepare | access
+        datasets/gov/example     |         |           |             |         |
+          | data                 | sql     |           |             |         |
+                                 |         |           |             |         |
+          |   |   | Country      |         |           | salis       |         |
+          |   |   |   | name[]   | string  |           | pavadinimas |         | open
+          |   |   |   | id       | integer required|           | id          |         | open
+        '''))
+
+    cli.invoke(rc, [
+        'copy',
+        '--no-source',
+        '--access', 'open',
+        '-o', tmp_path / 'result.mmd',
+              tmp_path / 'manifest.csv',
+    ])
+
+    with open(tmp_path / 'result.mmd', "r") as file:
+        contents = file.read()
+        assert contents == """---
+datasets/gov/example
+---
+classDiagram
+class Country {
++ name : string [0..*]
++ id : integer [1..1]
+}
+"""
+
+
+def test_copy_mmd_enum(context: Context, rc, cli: SpintaCliRunner, tmp_path):
+    create_tabular_manifest(context, tmp_path / 'manifest.csv', striptable('''
+        d | r | b | m | property | type    | ref       | source      | prepare | access
+        datasets/gov/example     |         |           |             |         |
+          | data                 | sql     |           |             |         |
+                                 |         |           |             |         |
+          |   |   | Country      |         |           | salis       |         |
+          |   |   |   | name     | string  |           | pavadinimas |         | open
+          |   |   |   | id       | integer required|           | id          |         | open
```

Comment:
You can fix column alignment using [Multiple cursors](https://www.jetbrains.com/help/pycharm/multicursor.html#toggle-between-normal-and-column-selection-modes) in PyCharm.

Reply:
O čia geras, ačiū :) Kažkada ieškojau, kaip tai taip paprastai padaryt, žinojau tik žymėjimą kelių vietų po vieną, su `Alt` ir kursorium

Reply:
sutvarkiau čia visur